index.js

```diff
@@ -6,38 +6,112 @@ const menu = ReadElement('#menu-icon');
 const nav = ReadElement('.mobile-nav');
 const WorkModal = ReadElement('.modal-section');
 const closeNav = ReadElement('.close-button');
-const viewModal = ReadElement('.view-modal');
-
-function AddMenu(selected, modalPart, opener) {
-  return selected.addEventListener('click', () => modalPart.classList.add(opener));
-}
+const viewModal1 = ReadElement('.view-modal1');
+const viewModal2 = ReadElement('.view-modal2');
+const viewModal3 = ReadElement('.view-modal3');
+const viewModal4 = ReadElement('.view-modal4');
 
 function RemoveMenu(selected, modalPart, remover) {
   return selected.addEventListener('click', () => modalPart.classList.remove(remover));
 }
 
+const details = [
+  {
+    title: 'Tonic',
```

Comment:
- [ OPTIONAL] I made this optional because I'm not sure at this point if is on my side that your images are not displaying when refreshing the page or is both sides, I think this could be the wrong path, KIndly fix this for your popup to look more professional and attractive :wink:  
![Screenshot from 2023-03-07 18-42-40](https://user-images.githubusercontent.com/88894525/223490778-91202021-26cf-4b1e-8f56-5307d64d8134.png)

Reply:
working fine